src/test/java/com/epam/tc/hw3/pages/components/SideMenuComponent.java

```diff
@@ -0,0 +1,15 @@
+package com.epam.tc.hw3.pages.components;
+
+import java.util.List;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class SideMenuComponent {
+
+    @FindBy(css = "ul[class='sidebar-menu left'] > li")
```

Comment:
Could be simplified to .sidebar-menu > li

Reply:
done